config/main.py

```diff
@@ -435,15 +447,25 @@ def del_vlan_member(ctx, vid, interface_name):
     vlan = db.get_entry('VLAN', vlan_name)
 
     if get_interface_mode() == "alias": 
-        interface_name = interface_alias(interface_name)
+        default_name = interface_alias_to_name(interface_name)
+        if default_name is None:
+            print "Invalid interface {}".format(interface_name)
+            sys.exit(0)
+        else:
+            interface_name = default_name
 
     if len(vlan) == 0:
         print "{} doesn't exist".format(vlan_name)
         raise click.Abort
     members = vlan.get('members', [])
     if interface_name not in members:
-        alias_name = interface_name_to_alias(interface_name)
         if get_interface_mode() == "alias": 
+            alias_name = interface_name_to_alias(interface_name)
+            if alias_name is None:
+                print "Invalid interface {}".format(interface_name)
+                sys.exit(0)
```

Comment:
Please replace `sys.exit(0)` `raise click.Abort()`, which will cause the program to exit with code 1.